apps/content/migrations/0037_alter_reciter_nationality_and_more.py

```
@@ -0,0 +1,30 @@
+        migrations.RemoveField(
+            model_name="reciter",
+            name="date_of_birth",
+        ),
+        migrations.RemoveField(
+            model_name="reciter",
+            name="is_contemporary",
+        ),
+        migrations.DeleteModel(
+            name="Nationality",
+        ),
+    ]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)